charts/astronomer/templates/houston/houston-configmap.yaml

```diff
@@ -68,61 +68,76 @@ data:
 
       # These values get passed directly into the airflow helm deployments
       helm:
-        # Webserver configuration.
-        webserver:
-          # This is the name of the secret that gets copied from the platform namespace
-          # to the airflow namespaces.
-          jwtSigningCertificateSecretName: {{ template "houston.jwtCertificateSecret" . }}
-          resources:
-            limits:
-              # XXX: There is an alert configured to trigger when ephemeral storage reaches
-              # 1800000000 bytes (90% of the 2Gi limit). Be aware of this when modifying
-              # ephemeral-storage limits.
-              ephemeral-storage: "2Gi"
-            requests:
-              ephemeral-storage: "1Gi"
-
-          # Disable creation of initial user.
-          defaultUser:
-            enabled: false
-    {{- if .Values.global.azure.enabled }}
-          livenessProbe:
-            failureThreshold: 25
-            periodSeconds: 10
-    {{- end }}
-
-        # Worker configuration (applies to Celery and Kubernetes task pods).
-        workers:
-          resources:
-            limits:
-              ephemeral-storage: "2Gi"
-            requests:
-              ephemeral-storage: "1Gi"
-
-          # This is here for upgrading to 0.11.1+ of Airflow.
-          keda:
-            enabled: false
-            namespaceLabels:
-              name: {{ .Release.Name }}
-
-          # This is here for upgrading to 0.11.2+ of Airflow.
-          safeToEvict: true
-
-        # Scheduler configuration.
-        scheduler:
-
-          # Scheduler pod disruption budget
-          podDisruptionBudget:
-            enabled: false
-
-            # PDB configuration
-            config:
-              maxUnavailable: 1
-
-          # This is here for upgrading to 0.11.2+ of Airflow.
-          safeToEvict: true
+      {{- if .Values.global.sccEnabled }}
+        # If security context constraints are enabled, enable SCCs for airflow-chart
+        sccEnabled: true
+      {{- end }}
+        airflow:
+          useAstroSecurityManager: true
+          webserver:
+            extraVolumeMounts:
```

Comment:
Do you think we should make a test for extraVolumeMounts, making sure that they do not appear when `Values.global.sccEnabled: False` ?

Reply:
the {{- if .Values.global.sccEnabled }} ends on line 74. So I expect these values to be here regardless of what sccEnabled is set to.